test/e2e/deployimage/plugin_cluster_test.go

```diff
@@ -279,6 +279,18 @@ func Run(kbc *utils.TestContext) {
 		}
 		return nil
 	}, time.Minute, time.Second).Should(Succeed())
+	By("getting memcached deploy by labels")
```

Comment:
Why we are adding this test? It is not required.

Reply:
Whay we need to do is change the label here: https://github.com/kubernetes-sigs/kubebuilder/blob/9a8fb397430b8ab191ed5f80db428da63e44dc61/test/e2e/deployimage/plugin_cluster_test.go#L255

Replace
`fmt.Sprintf("type=%s", strings.ToLower(kbc.Kind)),`

With

`fmt.Sprintf("app.kubernetes.io/name=%s", strings.ToLower(kbc.Kind)),`

It is the only change that we need on the tests.